### PR TITLE
document: filter results by org in admin view

### LINKED ISSUE #140: Results page: the facet "libraries" is missing in the professional View.
**Describe the issue**
Results page: the facet "libraries" is missing in the professional View. It exists in the public view.

**To Reproduce**
1. Go to professional view
2. Display the list of documents
3. There is no library facet

**Expected behavior**

**Software version**
rero-ils version https://github.com/rero/rero-ils/commit/c4dcb80048cf7e5c9ff81df815ee900e7d59286e
rero-ils-ui version 41f878c0

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by AoNoOokami**
* Fixes display of organisation facet in dcoument brief view.
* Modifies links to documents brief view to include current organisation as a default parameter.
* Closes #140.
* Removes useless code.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>
Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

https://tree.taiga.io/project/rero21-reroils/task/1377?kanban-status=1224895

## How to test?

Needs https://github.com/rero/rero-ils/pull/852 and last version of ng-core.
### Public search
1. Go to documents brief view
1. You should have an `organisation` facet in global view
1. You should have a `library` facet in organisation views.

### Admin
1. Go to documents brief view
1. The results should be filtered by current organisation by default (test with different users).
1. You should have a hierarchical facet (`organisation` + `libraries`)

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: projects/admin/src/app/routes/route-tool.service.ts [hunk 1]
```diff
@@ -214,7 +214,6 @@ export class RouteToolService {
   private aggFilter(aggregations: object) {
     const aggs = {};
     Object.keys(aggregations).map(aggregation => {
-      if ('organisation' !== aggregation) {
         if (aggregation.indexOf('__') > -1) {
```
Comment (sebastiendeleze) [inline projects/admin/src/app/routes/route-tool.service.ts:217]: Bad indentation

File: projects/admin/src/app/service/menu.service.ts [hunk 2]
```diff
@@ -90,12 +91,28 @@ export class MenuService {
...
   private _myLibraryRouterLink() {
     return `/records/libraries/detail/${this._userService.getCurrentUser().currentLibrary}`;
   }
+
+  /** Router link to my organisation */
+  private _myOrganisationRouterLink() {
+    return `/records/organisations/detail/${this._userService.getCurrentUser().library.organisation.pid}`;
+  }
+
+  /** Query params to filter documents by organisation */
```
Comment (sebastiendeleze) [inline projects/admin/src/app/service/menu.service.ts:114]: missing @return in doc blocks and return typing.

File: projects/admin/src/app/service/menu.service.ts [hunk 3]
```diff
@@ -90,12 +91,30 @@ export class MenuService {
...
 
+  /** Router link to my library */
   private _myLibraryRouterLink() {
     return `/records/libraries/detail/${this._userService.getCurrentUser().currentLibrary}`;
   }
+
+  /** Router link to my organisation */
+  private _myOrganisationRouterLink() {
+    return `/records/organisations/detail/${this._userService.getCurrentUser().library.organisation.pid}`;
+  }
```
Comment (sebastiendeleze) [inline projects/admin/src/app/service/menu.service.ts:112]: Please add @return for all methods

File: projects/admin/src/app/service/menu.service.ts [hunk 4]
```diff
@@ -90,12 +91,30 @@ export class MenuService {
...
+
+  /** Query params to filter documents by organisation
```
Comment (sebastiendeleze) [inline projects/admin/src/app/service/menu.service.ts:114]: Doc blocks must be written as this (empty line before and after method description)

```typescript
/**
 * Query params to filter documents by organisation
 *  
 * @return organisation pid as a dictionary
 */
```

**Comment by AoNoOokami** [comment]
> Some doc blocks are not well formatted (https://github.com/rero/rero-ils-ui/pull/208/files#diff-bf02d3d46f102054384e43e8deae3686R120, one empty line after description and @param and @return must be grouped), but it doesn't matter.

Ok, corrected. There was a typo in the commit message anyway.

**Comment by AoNoOokami** [comment]
Needs ng-core publication to pass the CI.

**Pull request merged**